models/PyCryptoBot.py

```diff
@@ -357,11 +357,13 @@ def getBuyMaxSize(self):
             return None
 
     def getDateFromISO8601Str(self, date: str) :
+		
         new_date_str = f'{date} 00:00:00' if len(date) == 10 else date
-        dt = new_date_str.split('T')
+																   
+        dt = new_date_str.split(' ')
         date = dt[0].split('-')
         time = dt[1].split(':')
-        return datetime(int(date[0]), int(date[1]), int(date[2]), int(time[0]), int(time[1]), int(time[2])) 
+        return datetime(int(date[0]), int(date[1]), int(date[2]), int(time[0]), int(time[1]), int(time[2]))
```

Comment:
is there any reason not to use `strptime()` here?

Reply:
I wasnt aware it would return a full datetime strptime sounds like it will remove the time.  I will make the change